proposals/83:upgrade-18/use.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Exit when any command fails
+set -uxeo pipefail
+
+# Econ Committee accept invitations for Committee and Charter
+./acceptInvites.js
+
+# "oracles" accept their invitations and provide prices to priceFeeds
+./verifyPushedPrice.js 'ATOM' 12.01
+./verifyPushedPrice.js 'stATOM' 12.01
+
+./openVault.js
```

Comment:
in the future we might want some vstorage snapshot tests to document the new state the subsequent tests can depend on. Out of scope, just thinking out loud